### PR TITLE
feat: Include live code sample i18n

### PR DESCRIPTION
**Pull request by marianfoo**
first example of a live code sample with CodeSandbox.io

Basically, all code is stored in a repository, independent of CodeSandbox. As an example, I first created one under my name, and later one should be created under DSAG.
CodeSandbox serves as an ad hoc runtime environment with which pull requests can also be made directly in branches.
Independently of this, the UI5 sample app can of course also be executed and changed locally.

The main branch can be used as a basis.
All other examples can be created in branches.
As an example, I have created an example for i18n.
Here I have embedded a CodeSandbox.
The embedding must point directly to the branch of the repository.
So as an example, here in this PR shows the embedding to:
_**https://codesandbox.io/embed/github/marianfoo/codesandbox-ui5-base/tree/i18n**_

```
<iframe src="https://codesandbox.io/embed/github/marianfoo/codesandbox-ui5-base/tree/i18n/?fontsize=14&module=%2Fsrc%2Fview%2FMainView.view.xml&theme=dark&view=editor"
     style="width:100%; height:500px; border:0; border-radius: 4px; overflow:hidden;"
     title="morning-thunder-dj6ol"
     allow="accelerometer; ambient-light-sensor; camera; encrypted-media; geolocation; gyroscope; hid; microphone; midi; payment; usb; vr; xr-spatial-tracking"
     sandbox="allow-forms allow-modals allow-popups allow-presentation allow-same-origin allow-scripts"
   ></iframe>
```

You can access a CodeSandbox instance directly from GitHub Branch if you write 'githubbox' instead of 'github', i.e.:
https://githubbox.com/marianfoo/codesandbox-ui5-base/tree/i18n
If you open this link, you can also directly use the "Fork" button to change the code and create a PR if necessary.

**So in summary:**
- Code is stored in GitHub regardless of provider
- CodeSandbox runs ad hoc code from GitHub (Branch)
- Examples can be differentiated into branches with a base example (main branch)
- Examples can be embedded directly, with live preview, button or link
- Examples from GitHub repository do not have to point to an existing instance on CodeSandbox.

What do you think?
@vobu @evilru @dscbs

### CODE REVIEW

**Comment by evilru** [comment]
The embedded code sandbox looks pretty good. Would it be possible to see the rendered output first like in the ui5 code examples?

Also, I have a feeling that the i18n code sample could be improved. This could be the wrong place to add comments to it - at least it shows that this also needs to be checked even if it is not added to the best practice repository ...

---

Mixed use of `var` and `let`. It should either use `const` and `let`, or if aiming for IE11 compatibility just use `var`.

Consistent formatting of the `sap.ui.define` part (Component.js, MainView.controller.js) would make sense. It is easier to read if each array entry has its own line, as in Component.js. (This would also make sense for the function parameters below).

I would prefer to call MainView.view.xml Main.view.xml.

SAP suggests [no longer using the Hungarion notation](https://blogs.sap.com/2021/02/17/why-the-hungarian-notation-should-not-be-used-for-modern-sapui5-development-anymore/).

According to the documentation, the i18n.property file should contain [additional information for translators](https://ui5.sap.com/#/topic/582ce93d326540f59d149031a44d5fb0). Note that this is only required for the source file.

In my opinion, it would be a good idea to explain why it's always good practice to name the i18n properties to describe their usage in a semantic way:

* home.positionTable.title
* overview.approveDialog.buttonCancel

**Comment by marianfoo** [comment]
Hi @evilru ,
you have three options on the embedding, i´ve chosen code because i thought that may be more relevant
image

Your feedback is very good.
I would also have to rewrite my i18n part a bit.
I have got used to the Hungarion notation, but the blog entry has good arguments for not using it any more.

I would suggest to create a new repo for the examples under the DSAG user.
There we can proceed with the review process in the same way as here.

**Comment by marianfoo** [comment]
Hi @evilru 
I have now created a new repo for the samples.
Each sample has its own branch and there is a base sample.
I´ve corrected the link in this commit to the new repo.

The improvements can then take place in this new repo.

I have already created a pull request for i18n and will incorporate your suggestions for improvement there.
https://github.com/1DSAG/UI5-Best-Practice-samples/pull/2

Can this be merged?

**Review by evilru — APPROVED** [review]
